apps/demo/partials/layout/base.hbs

```diff
@@ -46,6 +46,19 @@
         });
         targetElement.classList.add(`theme-${theme}`);
       };
+
+      // Hide elements with hide-from-visual-tests class in Supernova component preview via postMessage
```

Comment:
**nitpick (if-minor):** Can you please add a link to supernova.io settings so we can find a place where it is used? With a proper comment, of course.